en/yohane.md

```diff
@@ -2,33 +2,33 @@
 title: "Yohane Commands"
 old_id: 4
 ---
-Berikut ini adalah command-command yang ada di bot Yohane.
+These are the commands for the Yohane bot.
 
 ## Yohane In-Game
 
 ### General commands
-- `!roll [angka]` - Memberikan random number
-- `!help` - Memberikan daftar bantuan
-- `!subscribe pp [options]` - Untuk menampilkan status pp seperti limit dan notifikasi jika sudah selesai [submit score](https://cdn.discordapp.com/attachments/265909019976138754/801454622346444860/unknown.png). (options : `set`,`get`,`unset`,`toggle`)
-- `!boardshow [options]` - [Experimental] Untuk menyembunyikan diri dari leaderboard (options : `others`,`normal`)
-- `!report` - Jika kalian menemukan cheater, silahkan gunakan command ini.
-- `!faq list` - Daftar hal yang sering dibutuhkan dalam chat.
+- `!roll [number]` - Returns random number
+- `!help` - Shows a list of all commands
+- `!subscribe pp [options]` - Notifies about the pp status such as limit after [submitting a score](https://cdn.discordapp.com/attachments/265909019976138754/801454622346444860/unknown.png). (options: `set`,`get`,`unset`,`toggle`)
+- `!boardshow [options]` - [Experimental] Hide yourself from the leaderboard (options: `others`,`normal`)
+- `!report` - If you find a cheater, please use this command.
+- `!faq list` - Returns a list of things that are often needed in chat.
 
 ### Multiplayer commands
 
-Semua command yang ditampilkan disini hanya yang secara umum atau membutuhkan penjelasan. Mohon gunakan `!mp help` apabila bingung dengan command yang tidak terdaftar.
+These are general commands for the use in multiplayer rooms. Please use `!mp help` to show more commands.
```

Comment:
```suggestion
Commands that are shown here are for general use. Please use `!mp help` to show more commands.
```
translasi multiplayer nya dari mana?

Reply:
> translasi multiplayer nya dari mana?

belum di translate

Reply:
oh woop
s gw salah liat tadi

Reply:
Very "wonky" formulation. Will rephrase it for the sake of correct English. Also, please keep comments in English.

Reply:
Actually, the description is kinda redundant with the title and the list itself (as we mention the help twice in a row), we can just leave that out entirely imo.

Reply:
We could, since the title itself in self-explanatory

Reply:
Alright, will cut it in the next commit.